include/grpcpp/impl/codegen/client_interceptor.h

```diff
@@ -76,7 +76,7 @@ class ClientRpcInfo {
     UNKNOWN  // UNKNOWN is not API and will be removed later
   };
 
-  ~ClientRpcInfo(){};
+  ~ClientRpcInfo(){}
```

Comment:
(probably want to insert a space in front of `{}` here and elsewhere. clang-format omits it if the `;` is there but puts it there when it isn't)

Reply:
Done.